src/Components/WorkloadRules/WorkloadRules.js

```diff
@@ -57,10 +64,10 @@ const WorkloadRules = ({ workload }) => {
 
   const addFilterParam = (param, values) => {
     setExpandFirst(false);
-    return _addFilterParam(filters, updateFilters, param, values);
+    return workloadsRulesAddFilterParam(filters, updateFilters, param, values);
```

Comment:
Probably, we can remove return here?

```suggestion
    workloadsRulesAddFilterParam(filters, updateFilters, param, values);
```